### PR TITLE
fix: workflows folder name + add infra folder

### PR DESCRIPTION
**Pull request by agustibr**

### CODE REVIEW

**Comment by agustibr** [comment]
Hi! can we deploy this??

**Comment by eporter82** [comment]
This comment blocks by unresolved conversation?

File: README.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
 # plus_pull_replacement
-Hackdays@Devex code review 
+
+Hackdays@Devex code review
```
Comment (eporter82) [inline README.md:3]: Conversation test

**Review by eporter82 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (agustibr) [reply README.md:3]: Ok this conversation is blocking the automerge, I'll resolve it

File: README.md [hunk 2]
```diff
@@ -13,8 +16,7 @@ Hackdays@Devex code review
 - [Merging a pull request with a merge queue - GitHub Docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/merging-a-pull-request-with-a-merge-queue)
 - [About code owners - GitHub Docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)
 
-
-![Screenshot 2023-10-23 at 09-52-00 Settings · Branches · New branch protection rule · agustibr_plus_pull_replacement](https://github.com/agustibr/plus_pull_replacement/assets/210216/2817e547-c661-4c00-a289-7cf0ed013011)
+![Screenshot](https://github.com/agustibr/plus_pull_replacement/assets/210216/2817e547-c661-4c00-a289-7cf0ed013011)
 
 ---
```
Comment (agustibr) [inline README.md:21]: TBD, Add more info!

**Review by eporter82 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (agustibr) [reply README.md:21]: Will be added later, resolving

**Pull request merged**